node/_fs/_fs_opendir.ts

```diff
@@ -79,8 +79,16 @@ export function opendirSync(
 ): Dir {
   path = getValidatedPath(path).toString();
 
-  const encoding = options?.encoding ?? "utf8";
-  const bufferSize = options?.bufferSize ?? 32;
+  let encoding: string;
+  let bufferSize: number;
+  if (options) {
+    ({ encoding = "utf8", bufferSize = 32 } = options);
+  } else {
+    encoding = "utf8";
+    bufferSize = 32;
+  }
```

Comment:
Node uses a utility function called `getOptions()` here. We have that [here](https://github.com/denoland/deno_std/blob/3b9fb39ddd9a99e64aff159302039dc3cfddd36e/node/internal/fs/utils.mjs#L292) (although the second argument should be updated to default to `kEmptyObject`). I think it would be preferable to use that here.